checker/tests/nullness/Issue391.java

```diff
@@ -0,0 +1,50 @@
+// Test case for Issue 391:
+// https://github.com/typetools/checker-framework/issues/391
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.checker.nullness.qual.RequiresNonNull;
+import org.checkerframework.checker.nullness.qual.EnsuresNonNull;
+
+
+class ClassA {
+    @Nullable private String value = null;
```

Comment:
The type annotation on the return type should be directly in front of the return type.